Makefile

```diff
@@ -20,7 +20,7 @@ GOOS ?= $(shell go env GOOS)
 # The target architecture for linux kernel. Possible values: amd64 or arm64. 
 # Any other supported by linux kernel architecture could be added by introducing new build step into neonvm/hack/kernel/Dockerfile.kernel-builder
 KERNEL_TARGET_ARCH ?= amd64
-
+TARGET_ARCH ?= amd64
```

Comment:
Should we auto-detect this?

Reply:
Yes, I for sure will add as a part of the CI related issues (I mentioned them in a previous comment) while setting up arm builder!